src/main/java/org/jabref/logic/importer/fetcher/ComplexSearchQuery.java

```diff
@@ -134,14 +145,16 @@ public String toString() {
         getJournal().ifPresent(journal -> stringRepresentation.append(journal).append(" "));
         stringRepresentation.append(String.join(" ", getTitlePhrases()))
                             .append(String.join(" ", getDefaultFieldPhrases()))
-                            .append(String.join(" ", getAuthors()));
+                            .append(String.join(" ", getAuthors()))
```

Comment:
While you are on it, please switch to [StringJoiner](https://docs.oracle.com/javase/8/docs/api/java/util/StringJoiner.html)

```java
StringJoiner stringJoiner = new StringJoiner(" ");
getSingleYear().ifPresent(singleYear -> stringJoiner.append(singleYear));
...
```